src/cache.rs

```diff
@@ -692,7 +709,7 @@ impl Cache {
                 "cache::prepare(): expected source to be parsed before imports resolutions",
             )
         })?;
-        if import_res == CacheOp::Done(()) {
+        if matches!(import_res, CacheOp::Done(..)) {
```

Comment:
I think you can use a `if let CacheOp::Done(..) = import_res {`, which doesn't resort to a macro.